python/runtime/dbapi/maxcompute.py

```diff
@@ -11,12 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
-from odps import ODPS, tunnel
+try:
+    from odps import ODPS, tunnel
+    COMPRESS_ODPS_ZLIB = tunnel.CompressOption.CompressAlgorithm.ODPS_ZLIB
+except:  # noqa: E722
+    pass
```

Comment:
Can use `traceback.print_exc` so that when the job runs on PAI, we can know what was going on.

Reply:
Would do it in the next PR.